### DISCUSSION TITLE
Group-in-group level preset support

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#14184] [Suggest an Idea]
Today I came across the [Renovate documentation on Organization level presets](https://docs.renovatebot.com/config-presets/#organization-level-presets). It should be of great help for my organization, but I do see a caveat: It seems to be the case that Renovate is only expecting one level of groups, whereas the platform we use (Gitlab) alse supports groups-in-groups. If Renovate could also support that, there would be a more grannual support for specific presets for specific groups of projects. Was this ever considered?

**Comment by rarkins** [comment]
Wasn't considered. I guess we should check for a subgroup preset first and if not found then a group preset

**Reply by astellingwerf** [reply]
Yes, that's my assumption as well. I'd say we make it recursive, as even subgroups can have subgroups.
